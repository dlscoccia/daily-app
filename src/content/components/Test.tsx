const Test = () => {
  return <h2 className="text-lg text-red-500">Aja</h2>
}

export default Test
