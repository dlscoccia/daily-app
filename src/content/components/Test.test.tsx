import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'
import Test from './Test'

describe('Home', () => {
  it('renders a heading', () => {
    render(<Test />)

    const heading = screen.getByRole('heading', {
      name: /aja/i,
    })

    expect(heading).toBeInTheDocument()
  })

  it('renders homepage unchanged', () => {
    const { container } = render(<Test />)
    expect(container).toMatchSnapshot()
  })
})
