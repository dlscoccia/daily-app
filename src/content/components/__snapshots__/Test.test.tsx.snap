// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <h2
    class="aja"
  >
    Aja
  </h2>
</div>
`;
